### PR TITLE
Automatically send emails in case of errors.

### PR DESCRIPTION
**Pull request by vaibhavmagarwal**
In papertrails logs, we have to scroll up and down to see if an error has occurred. To save sometime, I have created a pulse_actions email bot account, which will send emails in case there is any error.

### CODE REVIEW

File: pulse_actions/email_config.cfg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "sender_list": ["vaibhavmagarwal@gmail.com"],
```
Comment (vaibhavmagarwal) [inline pulse_actions/email_config.cfg:2]: Right now, I have added my email id only. Let me know if any one else would like to subscribe to the errors.

**Comment by vaibhavmagarwal** [comment]
On the server, we will just need to change the password in the email_config.cfg file (can't send in the PR for obvious reasons).
r? @adusca @armenzg

**Comment by armenzg** [comment]
Papertrail allows you to set up alerts; have you looked into that?
These are searches which you can specify them to notify you. It's worked very well for me.

**Comment by vaibhavmagarwal** [comment]
Yes, papertrail worked! Thanks, closing this issue.

**Pull request closed without merging**